src/app/cli/src/init/mina_run.ml

```diff
@@ -464,7 +464,7 @@ let setup_local_server ?(client_trustlist = []) ?rest_server_port
                 , Snark_work_lib.Selector.Spec.Stable.Latest.to_yojson work.spec
                 )
               ] ;
-          log_snark_work_metrics work ;
+          (* log_snark_work_metrics work ; *)
```

Comment:
I leave these as comments on purpose, this is an intermediate state of our code and we should get rid of this comment as fast as possible